storage/tianmu/core/engine.cpp

```diff
@@ -635,7 +635,16 @@ std::shared_ptr<TableOption> Engine::GetTableOption(const std::string &table, TA
 
 void Engine::CreateTable(const std::string &table, TABLE *form) { TianmuTable::CreateNew(GetTableOption(table, form)); }
 
-AttributeTypeInfo Engine::GetAttrTypeInfo(const Field &field) {
+AttributeTypeInfo Engine::GetAttrTypeInfo(Field &field) {
+  AttributeTypeInfo ati = GetAttrTypeInfoInternal(field);
+  bitmap_set_bit(field.table->write_set, field.field_index);
+  bitmap_set_bit(field.table->read_set, field.field_index);
+  field.set_default();
+  ati.SetDefaultValue(&field);
```

Comment:
This line code was not covered by test.  but this line is the core code for this PR.

Reply:
The coverage code tool sometimes  does notn take effect, actually the cord code is runned,otherwise this PR does not work.